app/models/comfy/cms/page.rb

```diff
@@ -47,6 +48,10 @@ def self.in_category(category_id)
       .reorder('comfy_cms_categorizations.ordinal ASC')
   end
 
+  def translation(locale = :cy)
```

Comment:
yes, but as it stands you need to know what language you are currently in, so that you can ask for the other, It seems like it is complicating a method on the chance that there may be more languages in the future, or maybe it is just the default I have an issue with. Is there a way to make this work where you can ask a welsh page for the translation without having to explicitly pass :en?